index.html

```diff
@@ -52,18 +52,30 @@ <h1 class="sr-only">Bikes for Refugees Scotland - Providing donated bikes and ac
 			</div>
 			<div class="row">
 				<div class="primary-content col-lg-8">
-					<div class="jumbotron">
+					<div class="jumbotron col-lg-12">
+						<div class="back-contents col-lg-12">
```

Comment:
Be careful here, you are nesting `col-...` classes directly one inside the other. You cannot do this.

The order must be:

1) `container` (grandparent). 
You can use this just once to wrap around your whole webpage if you want, and then only have rows inside. 
2) `row` (parent). To group blocks together.
3) `col-...` (children) **note:** must add up to 12

Each level must be the direct child of the other.